regolith/helpers/l_grantshelper.py

```diff
@@ -67,7 +68,10 @@ def sout(self):
             desired_date = date_parser.parse(rc.date).date()
         else:
             desired_date = dt.date.today()
-        for grant in self.gtx["grants"]:
+        linked_grants = merge_collections(self.gtx["proposals"], self.gtx["grants"], "proposal_id")
+        count = 0
```

Comment:
where is count used or is it a debugging thing?  In which case please remove before merge